legion/engine/physics/colliders/convexcollider.cpp

```diff
@@ -43,12 +44,12 @@ namespace legion::physics
             manifold.isColliding = false;
             return;
         }
-     
+
         PointerEncapsulator < HalfEdgeFace> BRefFace;
         //log::debug("Face Check B");
         float BRefSeperation;
         if (PhysicsStatics::FindSeperatingAxisByExtremePointProjection(convexCollider,
-            this, manifold.transformA, manifold.transformB, BRefFace, BRefSeperation) || !BRefFace.ptr)
+            this, manifold.transformA, manifold.transformB, BRefFace, BRefSeperation, shouldDebug) || !BRefFace.ptr)
```

Comment:
I'm not a fan of the shouldDebug parameter for 'complete' code

Reply:
considering how volatile the physics engine is at the moment, I think its very likely I will need that again